lib/synapse_pay_rest/models/subnet/subnet.rb

```diff
@@ -116,6 +119,10 @@ def payload_for_create(node:, nickname:, **options)
           'nickname' => nickname
         }
         payload['account_class'] = options[:account_class] if options[:account_class].present?
+        if options[:card_type] == :prepaid
+          payload['account_type'] = PREPAID_ACCOUNT_TYPE
+          payload['wallet_style_id'] = PREPAID_WALLET_STYLE_ID
+        end
```

Comment:
Can you add a spec for this? These examples could work as reference 
* https://github.com/alice-financial/SynapsePayRest-Ruby/blob/alice/spec/models/user/user_spec.rb
* https://github.com/alice-financial/SynapsePayRest-Ruby/blob/alice/spec/models/shipment/shipment_spec.rb

Reply:
yes can do that this weekend